src/components/Shared/ProposalCountdown.tsx

```diff
@@ -50,7 +49,7 @@ export default class ProposalCountdown extends React.Component<IProps, IState> {
     this.stop();
   }
 
-  public calculateCountdown(endDate: Date | moment.Moment) {
+  private calculateCountdown(endDate: Date | moment.Moment) {
```

Comment:
How about using a single calculateCountdown  util function ?

Reply:
Done

Reply:
there is no commit

Reply:
check again

Reply:
https://github.com/daostack/alchemy/pull/1764/commits/e2cacc1f5d18bc16e3d264819e17f38f56374cb5